components/profile_popover/profile_popover.scss

```
@@ -4,3 +4,7 @@
     background: white;
     padding: 0
 }
+
+.bold {
```

Don't create utility CSS class names.
```suggestion
.user-profile-popover__heading {
```

Also, update the usages.